### PR TITLE
Supporting live dot coloring (by default tint is used)

### PR DESCRIPTION
**Pull request by AndriiMoskvin**
```
playerVC.customizeContentControlsProps = { props in
            guard case .player(let controls) = props else { return props }
            guard case .playable(let content) = controls.item else { return props }
            var newContent = content
            newContent.live.dotColor = .red
            
            return .player(.init {
                $0.playlist = controls.playlist
                $0.item = .playable(newContent)
                })
        }
```

screen shot 2017-09-11 at 6 34 52 pm
screen shot 2017-09-11 at 6 34 58 pm

### CODE REVIEW

**Comment by BogdanBilonog** [comment]
@AndriiMoskvin  Please, fix conflicts

File: PlayerControls/sources/ContentControlsUIProps.swift [hunk 1]
```diff
@@ -445,7 +446,15 @@ extension DefaultControlsViewController {
                 guard case .player(let player) = props else { return true }
                 guard case .playable(let props) = player.item else { return true }
                 
-                return props.isLive == false
+                return props.live.isHidden
+            }()
+            
+            liveDotColor = {
+                guard
+                    case .player(let player) = props,
```
Comment (BogdanBilonog) [inline PlayerControls/sources/ContentControlsUIProps.swift]: Better to use prism style `props.player?.foo.boo`

Comment (AndriiMoskvin) [reply PlayerControls/sources/ContentControlsUIProps.swift]: 👍

Comment (AndriiMoskvin) [reply PlayerControls/sources/ContentControlsUIProps.swift]: @BogdanBilonog Done.

**Pull request merged**